lang/po/Firmware_nl.po

```diff
@@ -0,0 +1,1787 @@
+msgstr "Er is een probleem opgetreden, Z-kalibratie afgedwongen ..."
+
+# MSG_SOUND_ONCE c=17 r=1
+#: sound.h:7
+msgid "Sound      [once]"
+msgstr "Geluid  [eenmaal]"
+
+# MSG_SOUND_SILENT c=17 r=1
+#: sound.h:8
+msgid "Sound    [silent]"
+msgstr "Geluid     [stil]"
+
+# MSG_SPEED
+#: ultralcd.cpp:7032
+msgid "Speed"
+msgstr "Snelheid"
+
+# MSG_SELFTEST_FAN_YES c=19
+#: messages.c:80
+msgid "Spinning"
+msgstr "Draait"
+
+# MSG_TEMP_CAL_WARNING c=20 r=4
+#: Marlin_main.cpp:4373
+msgid "Stable ambient temperature 21-26C is needed a rigid stand is required."
+msgstr "Stabiele omgevingstemperatuur van 21-26C is nodig, een stevige stand is vereist."
+
+# MSG_STATISTICS
+#: ultralcd.cpp:6945
+msgid "Statistics  "
+msgstr "Statistieken"
+
+# MSG_STOP_PRINT
+#: messages.c:93
+msgid "Stop print"
+msgstr "Print stoppen"
+
+# MSG_STOPPED
+#: messages.c:94
+msgid "STOPPED. "
+msgstr "GESTOPT. "
+
+# MSG_SUPPORT
+#: ultralcd.cpp:6954
+msgid "Support"
+msgstr ""
+
+# MSG_SELFTEST_SWAPPED
+#: ultralcd.cpp:8066
+msgid "Swapped"
+msgstr "Gewisseld"
+
+# MSG_TEMP_CALIBRATION c=20 r=1
+#: messages.c:95
+msgid "Temp. cal.          "
+msgstr "Tempkalibratie      "
+
+# MSG_TEMP_CALIBRATION_ON c=20 r=1
+#: ultralcd.cpp:5753
+msgid "Temp. cal.   [on]"
+msgstr "Tempkal.    [aan]"
+
+# MSG_TEMP_CALIBRATION_OFF c=20 r=1
+#: ultralcd.cpp:5751
+msgid "Temp. cal.  [off]"
+msgstr "Tempkal.    [uit]"
+
+# MSG_CALIBRATION_PINDA_MENU c=17 r=1
+#: ultralcd.cpp:5850
+msgid "Temp. calibration"
+msgstr "Tempkalibratie"
+
+# MSG_TEMP_CAL_FAILED c=20 r=8
+#: ultralcd.cpp:3948
+msgid "Temperature calibration failed"
+msgstr "Temperatuurkalibratie mislukt"
+
+# MSG_TEMP_CALIBRATION_DONE c=20 r=12
+#: messages.c:96
+msgid "Temperature calibration is finished and active. Temp. calibration can be disabled in menu Settings->Temp. cal."
+msgstr "Temperatuurkalibratie kan worden uitgeschakeld in het menu Instellingen-> Tempkalibratie."
+
+# MSG_TEMPERATURE
+#: ultralcd.cpp:5717
+msgid "Temperature"
+msgstr "Temperatuur"
+
+# MSG_MENU_TEMPERATURES c=15 r=1
+#: ultralcd.cpp:2235
+msgid "Temperatures"
+msgstr "Temperaturen"
+
+# MSG_FOLLOW_Z_CALIBRATION_FLOW c=20 r=4
+#: messages.c:42
+msgid "There is still a need to make Z calibration. Please follow the manual, chapter First steps, section Calibration flow."
+msgstr "Er is nog steeds een noodzaak om de Z-kalibratie uit te voeren. Volg de handleiding, hoofdstuk First steps, section Calibration flow."
+
+# 
+#: ultralcd.cpp:2901
+msgid "Total filament"
+msgstr "Totaal fil.   "
+
+# 
+#: ultralcd.cpp:2901
+msgid "Total print time"
+msgstr "Totaal printtijd"
+
+# MSG_TUNE
+#: ultralcd.cpp:6822
+msgid "Tune"
+msgstr "Fijnafstemming"
+
+# 
+#: 
+msgid "Unload"
+msgstr "Verwijderen"
+
+# 
+#: ultralcd.cpp:1804
+msgid "Total failures"
+msgstr "Totaal fouten"
+
+# 
+#: ultralcd.cpp:2308
+msgid "to load filament"
+msgstr "om filament te laden"
+
+# 
+#: ultralcd.cpp:2312
+msgid "to unload filament"
+msgstr "om filament te laden"
```

Comment:
typo, should be "om filament uit te werpen"

Reply:
Have to double check but i think this message is limited to c=20 r=1
Your translation is correct but is too long.

Reply:
"om fil. te ontladen" should fit, probably better